src/components/MultiStepForm/Steps/ExternalSupport.tsx

```diff
@@ -11,9 +11,19 @@ const externalSupportSchema = Yup.object({
 });
 
 export default function ExternalSupport() {
+	async function handleSubmit(values: Values) {
+		if (
+			values.supportType.includes("legal") &&
+			values.externalSupport === "yes"
```

Comment:
pensei na gente colocar esse if em uma const para dar mais contexto sobre essa condição, tipo:

```suggestion
			const hasGovernmentLegalSupport = values.supportType.includes("legal") && values.externalSupport === "yes"
```